diff -ruN hbmame-libretro.orig/Makefile.libretro hbmame-libretro/Makefile.libretro
--- hbmame-libretro.orig/Makefile.libretro	2021-02-02 09:38:46.818659903 -0300
+++ hbmame-libretro/Makefile.libretro	2021-02-02 09:39:46.455328985 -0300
@@ -9,7 +9,7 @@
 TARGET ?= hbmame
 # OS ?= linux
 # TARGETOS ?= linux
-PYTHON_EXECUTABLE ?= python2
+PYTHON_EXECUTABLE ?= python
 REGENIE ?= 1
 
 ###########################################################################
